src/ui/Search/frmsearchreplace.cpp

```diff
@@ -729,36 +730,49 @@ QStringList frmSearchReplace::fileFiltersFromUI()
     return filters;
 }
 
-void frmSearchReplace::addToHistory(QString string, QString type, QComboBox *comboBox)
-{
-    if (string != "") {
-        QSettings s;
-        QStringList history = s.value("Search/" + type + "History", QStringList()).toStringList();
-        history.prepend(string);
-        history.removeDuplicates();
-        history = history.mid(0, 10);
-        s.setValue("Search/" + type + "History", history);
-        comboBox->clear();
-        comboBox->addItems(history);
-    }
+void addToHistory(QStringList& history, QString string, QComboBox *comboBox) {
```

Comment:
Because it didn't really need to. It's pretty much just a small helper so I don't have to write the same piece of code 3x. It doesn't access any of `frmSearchReplace`'s data nor is it going to be of any interesting to any code outside of it.